### ISSUE TITLE
CSS

### ISSUE THREAD

**Issue by Axkaban** [renatav1/prj-rev-bwfs-dasmoto#2]
Awesome styles!

If you notice you have a few selectors that share the same `font-family` property, in those cases you may improve your efficiency by applying it once to one of the main global selectors like `*` (all selector)- which would overwrite any elements with sharing that property and trying to have a different value, therefore not very well liked by developers, but it does get to have its use cases, the `html` selector is a good option which also affects the title, or `body` selector which will share the property with all of its children and just like the `html` selector, it will respect if one or more of its children elements have different values in a shared property. The other option is multiple selections or group selection like so:
```
h1,
h2,
p {
  font-family: Helvetica;
}
```
Those options will help you save time and space on your file.

**Comment by renatav1** [comment]
Thanks again! So I want to be clear. The * selector selects everything?

**Comment by Axkaban** [comment]
That is correct. But it also keeps priority over any other value from other elements with the same property. That is why many people advice to never use it, although I believe that there is a use for everything at some point.
